app/views/hyrax/base/_items.html.erb

```diff
@@ -0,0 +1,31 @@
+<%# OVERRIDE: Hyrax v3.4.2 - remove table-striped class, remove <th> for visibility and actions for custom client theming of dc_repository & trace theme %>
```

Comment:
Great comment to help understand the "why" of this file.

I always shudder when we need to override views, but these comments really help ease my trepedations (the present implementation doesn't provide any better alternatives)